google/cloud/spanner/date.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_DATE_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_DATE_H_
+
+#include "google/cloud/spanner/version.h"
+#include <cstdint>
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+
+/**
+ * Represents a date in the proleptic Gregorian calendar.
+ *
+ * Standard C++11 lacks a good "date" abstraction, so we supply a simple
+ * implementation of a YMD triple. There is no range checking or field
+ * normalization. No calendar arithmetic is provided.
+ */
+class Date {
+ public:
+  Date() : Date(0, 0, 0) {}  // Note: bad month/day
+  Date(std::int64_t year, int month, int day)
+      : year_(year), month_(month), day_(day) {}
+
+  std::int64_t year() const { return year_; }
+  int month() const { return month_; }
```

Comment:
nit: I think we should document if this are zero-based or 1-based.

Reply:
Noted that month will be [1 .. 12] and day will be [1 .. 31].